independent-projects/tools/devtools-common/src/main/resources/codestarts/quarkus-extension/code/integration-test/java/integration-test/pom.tpl.qute.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+{#if it-parent}
+    <parent>
+        <groupId>{it-parent.group-id}</groupId>
+        <artifactId>{it-parent.artifact-id}</artifactId>
+        <version>{it-parent.version}</version>
+        {#if it-parent.relative-path}
+        <relativePath>{it-parent.relative-path}</relativePath>
+        {/if}
+    </parent>
+{/if}
+    <artifactId>{namespace.id}{extension.id}-integration-test</artifactId>
+{#if name.base && name.prefix}
+    <name>{name.prefix}{name.base}{name.segment-delimiter}Integration Test</name>
```

Comment:
The Quarkiverse extension is not generating a name for the `integration-test` module.

Reply:
ok this is a bug :) thanks

Reply:
good catch !